mmv1/third_party/terraform/resources/resource_dataproc_cluster.go.erb

```diff
@@ -638,6 +645,44 @@ func resourceDataprocCluster() *schema.Resource {
 											},
 										},
 									},
+
+									"reservation_affinity": {
+										Type:         schema.TypeList,
+										Optional:     true,
+										AtLeastOneOf: gceClusterConfigKeys,
+										Computed:     true,
+										MaxItems:     1,
+										Description:  `Reservation Affinity for consuming Zonal reservation.`,
+										Elem: &schema.Resource{
+											Schema: map[string]*schema.Schema{
+												"consume_reservation_type": {
+													Type:         schema.TypeString,
+													Optional:     true,
+													AtLeastOneOf: reservationAffinityKeys,
+													ForceNew:     true,
+													ValidateFunc: validation.StringInSlice([]string{"TYPE_UNSPECIFIED", "NO_RESERVATION", "ANY_RESERVATION", "SPECIFIC_RESERVATION"}, false),
```

Comment:
We typically don't need to include unspecified values like `TYPE_UNSPECIFIED`. Do you know if it is necessary here for some reason?

Reply:
Hi Ryan, from out proto, it does has this field. http://shortn/_BYbuhExsdX. 
I also followed similar example at line 683, which includes PREEMPTIBILITY_UNSPECIFIED. 

Let me know if you have concerns on this, i can remove it if you still think its not expected.